WordPress/Classes/Models/ReaderPost.h

```diff
@@ -34,6 +34,7 @@ extern NSString * const ReaderPostStoredCommentTextKey;
 @property (nonatomic) BOOL isReblogged;
 @property (nonatomic) BOOL isWPCom;
 @property (nonatomic) BOOL isSavedForLater;
+@property (nonatomic) BOOL isSeen;
```

Comment:
Maybe add a new entry in MIGRATIONS.txt about the new field?

Reply:
Ugh, I totally forgot. Thank you!